server/utils/visitorUtils.test.ts

```diff
@@ -32,7 +32,12 @@ describe('visitorUtils', () => {
       ['No restrictions', [], { isBanned: false }],
       ['No BAN restrictions', [TestData.restriction({ restrictionType: 'CLOSED' })], { isBanned: false }],
       [
-        'Single ban with no end date',
+        'Single ban with no end date (null)',
+        [TestData.restriction({ restrictionType: 'BAN', expiryDate: null })],
+        { isBanned: true },
+      ],
+      [
+        'Single ban with no end date (undefined)',
```

Comment:
Noticed that when we get data directly from the contact registry, visitor restrictions with no end date do not have `expiryDate` set (i.e. it is `undefined`). When we get the same data back from orchestration it would be `null`. Adding test for both scenarios so behaviour same wherever the data has come from.